Description: Updating python bindings for some changes in elm_toolbar
Author: otaviobp
Origin: upstream, http://trac.enlightenment.org/e/changeset/55183/trunk/BINDINGS/python/python-elementary
Forwarded: not-needed
Last-Update: 2011-08-18

Index: b/elementary/elementary.c_elementary_toolbar.pxi
===================================================================
--- a/elementary/elementary.c_elementary_toolbar.pxi
+++ b/elementary/elementary.c_elementary_toolbar.pxi
@@ -83,22 +83,13 @@
             raise ValueError("Object already deleted")
         elm_toolbar_item_del(self.obj)
 
-    def icon_name_get(self):
+    def icon_get(self):
         cdef const_char_ptr i
-        i = elm_toolbar_item_icon_name_get(self.obj)
+        i = elm_toolbar_item_icon_get(self.obj)
         if i == NULL:
             return None
         return i
 
-    property icon_name:
-        def __get__(self):
-            return self.icon_name_get()
-
-    def icon_get(self):
-        cdef c_evas.Evas_Object *icon
-        icon = elm_toolbar_item_icon_get(self.obj)
-        return evas.c_evas._Object_from_instance(<long> icon)
-
     property icon:
         def __get__(self):
             return self.icon_get()
Index: b/include/elementary/c_elementary.pxd
===================================================================
--- a/include/elementary/c_elementary.pxd
+++ b/include/elementary/c_elementary.pxd
@@ -535,8 +535,7 @@
     Elm_Toolbar_Item *elm_toolbar_last_item_get(evas.c_evas.Evas_Object *obj)
     Elm_Toolbar_Item *elm_toolbar_item_next_get(Elm_Toolbar_Item *item)
     Elm_Toolbar_Item *elm_toolbar_item_prev_get(Elm_Toolbar_Item *item)
-    evas.c_evas.Evas_Object *elm_toolbar_item_icon_get(Elm_Toolbar_Item *item)
-    char *elm_toolbar_item_icon_name_get(Elm_Toolbar_Item *item)
+    char *elm_toolbar_item_icon_get(Elm_Toolbar_Item *item)
     char *elm_toolbar_item_label_get(Elm_Toolbar_Item *item)
     void  elm_toolbar_item_label_set(Elm_Toolbar_Item *item, char *label)
     void *elm_toolbar_item_data_get(Elm_Toolbar_Item *item)
